### PR TITLE
Adding the button using JS and styling with CSS and Image [ #48277 ]

### PR DESCRIPTION
**Pull request by ally1002**
### Motivation / Background

At this pull request I implemented a back-to-top button on guides

### Detail

guides/assets/javascripts/guides.js
guides/assets/stylesheets/main.css
guides/assets/images/up_white_arrow.png
guides/CHANGELOG.md

### Checklist

Before submitting the PR make sure the following are checked:

* [x] This Pull Request is related to one change. Changes that are unrelated should be opened in separate PRs.
* [x] Commit message has a detailed description of what changed and why. If this PR fixes a related issue include it in the commit message. Ex: `[ #48277 ]`
* [x] Tests are added or updated if you fix a bug or add a feature.
* [x] CHANGELOG files are updated for the changed libraries if there is a behavior change or additional feature. Minor bug fixes and documentation changes should not be included.

### CODE REVIEW

**Comment by ally1002** [comment]
image

**Comment by ally1002** [comment]
#48277 Issue

File: guides/CHANGELOG.md [hunk 1]
```diff
@@ -1 +1,3 @@
 Please check [7-0-stable](https://github.com/rails/rails/blob/7-0-stable/guides/CHANGELOG.md) for previous changes.
+
+*   Add "back to top" button to guides
```
Comment (guilleiguaran) [inline guides/CHANGELOG.md:3]: The new entries in the CHANGELOG are added at the top of the file.

**Comment by guilleiguaran** [comment]
One last thing, please squash your commits in a single one

**Comment by p8** [comment]
@ally1002 The arrow image isn't currently showing as the css uses dashes (`images/up-white-arrow.png`) but the file uses underscores (`images/up_white_arrow.png`).

**Comment by p8** [comment]
No worries @ally1002 Thanks for the fix 😄

**Comment by p8** [comment]
@ally1002 You might be interested in a similar issue in the API docs:
https://github.com/rails/sdoc/issues/209

**Pull request merged**